src/bin/huak/commands/activate.rs

```diff
@@ -1,6 +1,28 @@
-use crate::errors::CliResult;
+use crate::errors::{CliError, CliResult};
+use std::env;
+use std::process::ExitCode;
+
+use huak::project::Project;
 
 /// Run the `activate` command.
 pub fn run() -> CliResult<()> {
-    unimplemented!()
+    let cwd = env::current_dir()?;
+    let project = match Project::from(cwd) {
+        Ok(p) => p,
+        Err(e) => return Err(CliError::new(e, ExitCode::FAILURE)),
+    };
+
+    let venv = project
+        .venv()
+        .as_ref()
+        .expect("`Project::from` creates venv if it doesn't exists.");
+
+    println!("Venv activated: {}", venv.path.display());
+
+    venv.activate()
+        .map_err(|e| CliError::new(e, ExitCode::FAILURE))?;
+
+    println!("Venv deactivated.");
```

Comment:
So the pattern for implementations we're going with right now is to throw operations we perform on projects in huak::ops. Could we move this there? This allows us to use the bin space for more CLI-related handling and logic.

Reply:
Can do!